.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: required 
+dist: trusty 
```

Comment:
Travis doesn't support 16.04, so tests for 16.04 are run in a Docker container. The host can then run trusty.